hypgraph.cpp

```diff
@@ -1581,7 +1582,10 @@ EX void draw_model_elements() {
         }
 
       queuereset(pmodel, PPR::CIRCLE);
-      /* fallthrough */
+      ld a = -models::model_orientation * degree;
+      queuestr(xspinpush0(a, +vid.twopoint_param), vid.xres / 100, "X", ringcolor >> 8);
+      queuestr(xspinpush0(a, -vid.twopoint_param), vid.xres / 100, "X", ringcolor >> 8);
+      return;

```

Comment:
Here I just duplicated the code from the following `case`. Other options would be:

- `goto fallthru; fallthru:` as I used above (but I don't recommend this because it connotes ugly hackery)

- factor the shared code out into a function and call that function from both places